components/navbars.py

```diff
@@ -35,49 +39,51 @@ def is_logged_out(self):
+    meetings_link = Locator(By.CSS_SELECTOR, 'ul.dropdown-menu.service-dropdown > li:nth-child(4) > a')
+    institutions_link = Locator(By.CSS_SELECTOR, 'ul.dropdown-menu.service-dropdown > li:nth-child(5) > a')
 
-class EmberNavbar(HomeNavbar):
-    user_dropdown = Locator(By.CSS_SELECTOR, 'ul.navbar-nav > li:nth-child(6) > a')
-    user_dropdown_profile = Locator(By.CSS_SELECTOR, 'ul.auth-dropdown > li:nth-child(1)')
-    user_dropdown_support = Locator(By.CSS_SELECTOR, 'ul.auth-dropdown > li:nth-child(2)')
-    user_dropdown_settings = Locator(By.CSS_SELECTOR, 'a[data-analytics-name="Settings"]')
-    logout_link = Locator(By.CSS_SELECTOR, '[data-test-ad-logout]')
+    my_preprints_link = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li:nth-last-child(6) > a')
+    add_a_preprint_link = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li:nth-last-child(5) > a')
+    search_link = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li:nth-last-child(4) > a')
+    support_link = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li:nth-last-child(3) > a')
+    donate_link = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li.navbar-donate-button')
+    sign_up_button = Locator(By.CSS_SELECTOR, 'a.btn-success:nth-child(1)')
     sign_in_button = Locator(By.CSS_SELECTOR, '.btn-top-login')
-    donate_link = Locator(By.XPATH, '//a[text()="Donate"]')
 
-
-class PreprintsNavbar(AbstractLegacyEmberNavbar):
-    title = Locator(By.CSS_SELECTOR, '.navbar-title')
-    add_a_preprint_link = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li:nth-last-child(5) > a')
+    user_dropdown_profile = Locator(By.CSS_SELECTOR, 'ul.dropdown-menu-right > li:nth-child(1)')
+    user_dropdown_support = Locator(By.CSS_SELECTOR, 'ul.dropdown-menu-right > li:nth-child(2)')
+    user_dropdown_settings = Locator(By.CSS_SELECTOR, 'ul.dropdown-menu-right > li:nth-child(3)')
+    logout_link = Locator(By.CSS_SELECTOR, '#secondary-navigation > ul > li.dropdown.open > ul > li:nth-child(4) > a')
 
     def verify(self):
         return self.current_service.text == 'PREPRINTS'
 
 
 class RegistriesNavbar(EmberNavbar):
+    home_link = Locator(By.CSS_SELECTOR, 'ul._ServiceDropdownMenu_nar5mu > li:nth-child(1) > a')
+    preprints_link = Locator(By.CSS_SELECTOR, 'ul._ServiceDropdownMenu_nar5mu > li:nth-child(2) > a')
+    registries_link = Locator(By.CSS_SELECTOR, 'ul._ServiceDropdownMenu_nar5mu > li:nth-child(3) > a')
+    meetings_link = Locator(By.CSS_SELECTOR, 'ul._ServiceDropdownMenu_nar5mu > li:nth-child(4) > a')
+    institutions_link = Locator(By.CSS_SELECTOR, 'ul._ServiceDropdownMenu_nar5mu > li:nth-child(5) > a')
+
+    help_link = Locator(By.CSS_SELECTOR, 'a[data-test-help')
+    donate_link = Locator(By.CSS_SELECTOR, 'a[data-test-donate')      
+    join_link = Locator(By.CSS_SELECTOR, 'a[data-test-join')
+    login_link = Locator(By.CSS_SELECTOR, 'a[data-test-login')
```

Comment:
The above four selectors are missing the final `]` on the selector.

Reply:
Corrected in latest commit.